model/textparse/promparse_test.go

```diff
@@ -492,7 +492,7 @@ const (
 	promtestdataSampleCount = 410
 )
 
-func BenchmarkParse(b *testing.B) {
+func BenchmarkPromParse(b *testing.B) {
 	for parserName, parser := range map[string]func([]byte, *labels.SymbolTable) Parser{
 		"prometheus": NewPromParser,
 		"openmetrics": func(b []byte, st *labels.SymbolTable) Parser {
```

Comment:
Does this even makes sense? Why we use openmetrics parser for `"promtestdata.txt", "promtestdata.nometa.txt"`?

Reply:
🤷
OM parser was already here but in a comment above Arthur mentioned that Prom text is a subset of OM so it works out

Reply:
Interesting. Is this something that users are doing (parsing PromText with OpenMetrics text)?

Reply:
I guess they were or someone wanted to compare it, ack! thx